js/src/util/index.js

```diff
@@ -261,6 +261,34 @@ const execute = callback => {
   }
 }
 
+/**
+ * Return the previous/next element of a list.
+ *
+ * @param {array} list    The list of elements
+ * @param activeElement   The active element
+ * @param shouldGetNext   Choose to get next or previous element
+ * @param {bool} isCycleAllowed
+ * @return {Element|elem} The proper element
+ */
+const pickFromList = (list, activeElement = null, shouldGetNext = true, isCycleAllowed = true) => {
```

Comment:
pickFromList should be renamed here and everywhere else too, right?

BTW, if we don't need default values for all variables, we should skip them.